docs/user-guide/garbage-collection.md

```diff
@@ -0,0 +1,84 @@
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/user-guide/garbage-collection.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Garbage Collection
+
+Garbage collection is managed by kubelet automatically, mainly including unreferenced
+images and dead containers. kubelet applies container garbage collection every minute
+and image garbage collection every 5 minutes.
+
+### Image Collection
+
+kubernetes manages lifecycle of all images through `imageManager`。
+The policy for garbage collecting images we apply takes two factors into consideration,
+`HighThresholdPercent` and `LowThresholdPercent`. Disk usage above the the high threshold
+will trigger garbage collection,  and never down to low threshold.
+
+### Container Collection
+
+The policy for garbage collecting containers we apply takes on three variables, which can
+be user-defined. See also [here](#User Configuration).
+kubelet sorts out containers which are unidentified or stay out of bounds set by previous
+mentioned three flags. For the latter case, the oldest containers are removed first.
+When kubelet cleans up the dead containers, the associated volumes will be taken care of as well.
+
+> Note that we will skip the containers that are not managed by kubelet.
+
+### User Configuration
+
+Users are free to set their own value to address image garbage collection.
+
+- `image-gc-high-threshold`, the percent of disk usage after which image garbage collection is
+always run. Default is 90%
+- `image-gc-low-threshold`, the percent of disk usage before which image garbage collection is
+never run. Lowest disk usage to garbage collect to. Default is 80%.
+
+We also allow users to customize garbage collection policy, basically via following three flags.
+
+- `minimum-container-ttl-duration`, minimum age for a finished container before it is garbage collected.
+- `maximum-dead-containers-per-container`, maximum number of old instances of a container to retain
+per container. Each container takes up some disk space.
+- `maximum-dead-containers`, maximum number of old instances of a containers to retain globally. Each
+container takes up some disk space.
+Value less than zero for the last two is regarded as no limit.
```

Comment:
super nit:  I believe these should be plural e.g.
Values less than zero for the last two are regarded as no limit.